### ISSUE TITLE
3.1.1 missing types

### ISSUE THREAD

**Issue by jsw** [harpreetkhalsagtbit/country-state-city#118]
This import works in 3.0.1 but not 3.1.0

```
import { Country, State } from 'country-state-city';
```

```
Type error: Could not find a declaration file for module 'country-state-city'
```

I notice there is no longer a `dist/index.d.ts` as referenced in `package.json`.
The file has been moved to `lib` in 3.1.0.

**Comment by jsw** [comment]
I see it's been fixed in 3.1.1. Thanks! I didn't see the release on github.

**Comment by jsw** [comment]
Actually, it appears 3.1.1 has the same issue. There is no dist/index.d.ts in the distribution.

**Comment by harpreetkhalsagtbit** [comment]
@jsw 
thanks, I actually updated the master but seems like somehow it didn't get published properly

**Comment by harpreetkhalsagtbit** [comment]
@jsw just published `3.1.2` - Please check.

**Comment by jsw** [comment]
`3.1.2` works. Thanks for the quick turnaround.
